builder/scaleway/config.go

```diff
@@ -67,7 +67,7 @@ type Config struct {
 
 	RemoveVolume bool `mapstructure:"remove_volume"`
 
-	UserAgent string
+	UserAgent string `mapstructure:"user_agent"`
```

Comment:
I think for this case is more appropriate if you set the mapstructure like 
https://github.com/hashicorp/packer/blob/master/builder/azure/arm/config.go#L410

```suggestion
	UserAgent string `mapstructure-to-hcl2:",skip"`
```

Cuz this is not actually a config option, is just a runtime variable.